doc/man7/provider.pod

```diff
@@ -204,6 +227,15 @@ an implementation.
 The method object that is fetched can then be used with diverse other
 functions that use them, for example L<EVP_DigestInit_ex(3)>.
 
+Operations that work with asymmetric keys via an B<EVP_PKEY_CTX> have a slightly
+different form. In those cases the fetch of the algorithm is done internally
```

Comment:
Is there a difference between "internally" and "implicitly" here?  I view it as an implicit fetch, the only difference being that there's currently (*) no way to pass an explicitly fetched EVP_KEYMGMT.

(*) we *could* have a `EVP_PKEY_CTX_new_from_keymgmt()`...  I don't currently see an actual need, and it kinda defeats the purpose with how EVP_PKEY_CTX is created, even with legacy methods, as there's no way to create one from an `EVP_PKEY_METHOD` or an `EVP_PKEY_ASN1_METHOD` either...

Reply:
Ok, that argument works for me
(as for new terms, if you still want to toy with that, how about "demi-explicit"? :wink:)